app/controllers/users_controller.rb

```diff
@@ -1,5 +1,7 @@
 class UsersController < ApplicationController
```

Comment:
cái này chị nghĩ là của role admin nhỉ, tìm hiểu về `namespace` đi bé, sáng to quên chỉ to php